wordpress-mobile/WordPress-Android
# Create video plays block in Stats

Fixes #8682 
FluxC PR needs to be merged with this PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1036

This PR is adding a Video plays block to the Day/Week/Month/Year stats. The block contains a list of videos. The videos are clickable and redirect to the post with the video. Next to the video titles there is a play count for that video. The View more button should be visible when there are 6+ videos.  

To test:
* Open Stats
* Go to Days/Weeks/Months/Years tab
* You see a Videos block at the bottom of the screen
* Videos in the block are clickable and redirect to the Post with the video
* There is a `View more` button when there are more than 6 videos
* The `View more` button redirects to the View all page in the old designs
